cypress/support/pages/room_administration/pageRoomAdmin.js

```diff
@@ -0,0 +1,32 @@
+class RoomAdmin {
+	static #submenuRoomAdminLink = '[data-testid="sidebar-management-rooms"]';
```

Comment:
this selector has never been used in the file

Reply:
why this is used? its used in this method in this file.

navigateToRoomsAdministrationPageViaSubmenu() {
		cy.get(RoomsAdmin.#submenuRoomAdminLink).should("be.visible").click();
	}